src/cpr_data_access/models.py

```diff
@@ -196,14 +196,19 @@ def to_string(self) -> str:
         """Return text in a clean format"""
         return " ".join([line.strip() for line in self.text])
 
+    def __hash__(self) -> int:
+        """Get hash of the text-block. Based on the text and the text_block_id"""
+        text_utf8 = self.to_string().encode("utf-8")
+
+        return hash(f"{text_utf8}-{self.text_block_id.encode()}")
```

Comment:
is it worth returning a string of the hash here as it's likely to be printed/saved, and this matches the behaviour of the other hash function? no strong opinions though so feel free to ignore

Reply:
Unfortunately the `__hash__()` magic method enforces int return type, so we can't return a string